algoliasearch-common/src/main/java/com/algolia/search/http/AlgoliaRequestKind.java

```diff
@@ -0,0 +1,7 @@
+package com.algolia.search.http;
+
+public enum AlgoliaRequestKind {
+  SEARCH,
```

Comment:
I would prefix the values by the product.
So here:
`SEARCH_API_SEARCH`
`SEARCH_API_WRITE`
`ANALYTICS_READ`

Reply:
Good idea, changing it.